apis/contour/v1beta1/ingressroute.go

```diff
@@ -77,6 +77,14 @@ type Delegate struct {
 	Namespace string `json:"namespace"`
 }
 
+// Status reports the current state of the IngressRoute
+type Status struct {
+	CurrentStatus   string      `json:"currentStatus"`
+	Description     string      `json:"description"`
+	LastProcessTime metav1.Time `json:"lastProcessTime"`
```

Comment:
I had a spelunk in the other k8s objects looking for a theme for this field name and couldn't find a pattern. Indeed I couldn't find anything that included a timestamp in status. Even pod doesn't include the timestamp when it last updated the field, it includes a timestamp when the condition change, see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#containerstate-v1-core

I'm wondering if this is a hint that we may be making a mistake. Including a timestamp feels like a safety blanket that if the last processed time is too old, maybe we distrust its value.

I'd like to suggest dropping last processed time for the beta.